sockets/delegateMonitor.js

```diff
@@ -326,55 +312,57 @@ module.exports = function (app, connectionHandler, socket) {
 				data.votes = res[2];
 				data.nextForgers = cutNextForgers(10);
 
-				log('info', 'Emitting data');
+				log('debug', 'Emitting data');
 				socket.emit('data', data);
 			}
 		});
 	};
 
 	this.onInit = function () {
 		this.onConnect();
-
-		async.parallel([
-			// We only call getLastBlock on init, later data.lastBlock will be updated from getLastBlocks
-			getLastBlock,
-			getActive,
-			getRegistrations,
-			getVotes,
-			getNextForgers,
-		],
-		(err, res) => {
-			if (err) {
-				log('error', `Error retrieving: ${err}`);
-			} else {
-				tmpData.nextForgers = res[4];
-
-				data.lastBlock = res[0];
-				data.active = updateActive(res[1]);
-				data.registrations = res[2];
-				data.votes = res[3];
-				data.nextForgers = cutNextForgers(10);
-
-				log('info', 'Emitting new data');
-				socket.emit('data', data);
-
-				getLastBlocks(data.active, true);
-
-				newInterval(0, 5000, emitData);
-				newInterval(1, 1000, getLastBlocks);
-			}
-		});
 	};
 
 	this.onConnect = function () {
-		log('info', 'Emitting existing data');
+		log('debug', 'Emitting existing data');
 		socket.emit('data', data);
 	};
 
 	this.onDisconnect = function () {
-		for (let i = 0; i < intervals.length; i++) {
-			clearInterval(intervals[i]);
-		}
-		intervals = [];
+		log('debug', 'Client disconnected');
 	};
+
+	async.parallel([
+		// We only call getLastBlock on init, later data.lastBlock will be updated from getLastBlocks
+		getLastBlock,
+		getActive,
+		getRegistrations,
+		getVotes,
+		getNextForgers,
```

Comment:
`getNextForgers` should be updated in an independent loop, as `blocks/change`/`rounds/change` events will not trigger if the slot is missed.

Reply:
Done. Implemented additional interval-based updates when the block time is exceeded.